components/src/dynamo/planner/tests/unit/test_diagnostics_recorder.py

```
@@ -0,0 +1,303 @@
+            osl=osl,
+            request_duration=2.5,
+        )
+
+        data.append((tick_input, effects, observed, gpu_hours))
+
+    return data
+
+
+class TestDiagnosticsRecorder:
+    def test_disabled_when_no_interval(self):
+        cfg = _make_config("/tmp", report_interval_hours=None)
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)